softlayer/resource_softlayer_block_storage.go

```diff
@@ -101,7 +101,7 @@ func resourceSoftLayerBlockStorage() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
-						"hostIQN": {
```

Comment:
was this change needed as well?

Reply:
yes, this is required. There is a check in terraform that make sure all parameter name is lower case.

Reply:
Ok.